[package] qt/5.15.4: -o qt:with_dbus=True results in fatal error LNK1104: cannot open file dbus-1d.lib on Windows

### Package and Environment Details

* Package Name/Version: **qt/5.15.4** using **dbus/1.12.20**
* Operating System+version: **Windows 10 64-bit**
* Compiler+version: **Visual C++ 2019 16.11.16**
* Conan version: **conan 1.49.0**
* Python version: **Python 3.10.4**


### Conan profile

```ini
[settings]
os=Windows
os_build=Windows
arch=x86_64
arch_build=x86_64
compiler=Visual Studio
compiler.version=16
build_type=Release
[options]
[conf]
[build_requires]
[env]
```

### Steps to reproduce

In a debug build with qt:with_dbus=True, Qt looks for dbus-1d.lib, but the debug build of dbus on Windows produces dbus-1.lib without the d suffix.

To reproduce run

```
conan install qt/5.15.4@ -o qt:with_dbus=True -s build_type=Debug --build
```

Expected result:
Successful build

Actual result:
Build fails with fatal linker error:

```
linking ..\..\..\bin\qdbuscpp2xml.exe
LINK : fatal error LNK1104: cannot open file 'dbus-1d.lib'
``` 


### Logs

_No response_

--- a/recipes/qt/5.x.x/conanfile.py
+++ b/recipes/qt/5.x.x/conanfile.py
@@ -420,6 +420,10 @@ class QtConan(ConanFile):
                                   "  if (enable_precompiled_headers) {\n    if (is_win) {",
                                   "  if (enable_precompiled_headers) {\n    if (false) {"
                                   )
+        tools.replace_in_file(os.path.join(self.source_folder, "qt5", "qtbase", "configure.json"),
+                                  "-ldbus-1d",
+                                  "-ldbus-1"
+                                  )
 
     def _make_program(self):
         if self._is_msvc:

